dockerbuild/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM archlinux
+
+RUN pacman -Syy --noconfirm \
+	community/nds32le-elf-binutils \
```

Comment:
No need to install anything from the repos, you can install the packages directly. Also, this command does a partial upgrade, which is not supported, should do `pacman -Syu` instead.

Reply:
you seem to be wrong here:
```
Sending build context to Docker daemon  3.072kB
Step 1/6 : FROM archlinux
 ---> c24fe13d37b9
Step 2/6 : ENV GCC=nds32le-elf-gcc-9.3.0-1-x86_64.pkg.tar.xz
 ---> Using cache
 ---> 889db3b11494
Step 3/6 : ENV NEWLIB=nds32le-elf-newlib-3.3.0-1-any.pkg.tar.xz
 ---> Using cache
 ---> e494279d2227
Step 4/6 : RUN curl https://github.com/kittennbfive/lkv373-tools/releases/download/0.01/$GCC -L -O
 ---> Using cache
 ---> d0c8209853a2
Step 5/6 : RUN curl https://github.com/kittennbfive/lkv373-tools/releases/download/0.01/$NEWLIB -L -O
 ---> Using cache
 ---> 8ee2a2cbecb2
Step 6/6 : RUN pacman -U $GCC $NEWLIB
 ---> Running in d3e5f5e91bdf
warning: database file for 'core' does not exist (use '-Sy' to download)
warning: database file for 'extra' does not exist (use '-Sy' to download)
warning: database file for 'community' does not exist (use '-Sy' to download)
loading packages...
resolving dependencies...
warning: cannot resolve "nds32le-elf-binutils", a dependency of "nds32le-elf-gcc"
warning: cannot resolve "libmpc", a dependency of "nds32le-elf-gcc"
:: Do you want to skip the above package for this upgrade? [y/N] error: failed to prepare transaction (could not satisfy dependencies)
:: The following package cannot be upgraded due to unresolvable dependencies:
      nds32le-elf-gcc

:: unable to satisfy dependency 'nds32le-elf-binutils' required by nds32le-elf-gcc
:: unable to satisfy dependency 'libmpc' required by nds32le-elf-gcc
The command '/bin/sh -c pacman -U $GCC $NEWLIB' returned a non-zero code: 1
```
what do you suggest?

Reply:
Ah sorry, it is binutils. 

Change the line to
```
pacman -Syu --needed --noconfirm nds32le-elf-binutils
```